RELEASE_NOTES.md

```diff
@@ -120,22 +122,22 @@ accompanying blog post.
 
 - Try to get color-space for PNG file from the cICP chunk. This was
   added in a recent revision of the PNG spec, so we take advantage of
-  it is present.
+  it if present.
 
 - The reading of 16-bit half float for TIFF format is now done using
   the Imath library.
 
 - Enable actions on the export format and storage widgets. This means
-  that they can be mapped to shortcut.
+  that they can be mapped to shortcuts.
 
 - Introduce balanced OpenCL vs CPU tiling. This makes it possible to
-  use CPU tiling if there is not enough memory on the OpenCL card and
+  use CPU tiling, if there is not enough memory on the OpenCL card and
   so would require a lot of tiles to be handled on the card. At the
   end, the large number of tiles plus the overlapping area will make
   the use of the OpenCL code patch slower (or much slower) than
   handling the image without tiling on CPU.
```

Comment:
Perhaps:
Introduce balanced OpenCL vs CPU tiling. This makes it possible to
use CPU tiling, if there is not enough memory available on the card for OpenCL.
Such a setup / situation would lead to excessive tiling. In the end, the large number of tiles,
plus the overlapping area, would make the use of the OpenCL code patch (much) slower than
handling the image without tiling on CPU.

Reply:
Too many commas I think, but I'll look at it in the round later.